<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScreen</class>
 <widget class="QDialog" name="RegisterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterScreen{
background-color: #2a6496;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="title_label">
             <property name="font">
              <font>
               <family>Script MT Bold</family>
               <pointsize>24</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Student Information system</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="info_frame">
        <property name="styleSheet">
         <string notr="true">/* Style the info_frame element */
#info_frame {
    background-color: #fff;
    border: none;
    border-radius: 5px;
}

#info_frame #radioframe{
     border: 1px solid #838383;
     border-radius: 5px;
}

#info_frame #Imagelabel{
background-color: #f0ad4e;
}
/* Style for labels, input fields, and combo boxes inside &quot;info_frame&quot; */
#info_frame QLabel,
#info_frame QLineEdit,
#info_frame QComboBox,
QHeaderView::section {
    font-family: Segoe UI Semibold;
    font-size: 12px;
}

/* Style for input fields and combo boxes inside info_frame */
#info_frame QLineEdit,
#info_frame QComboBox {
    padding: 4px 5px; /* Corrected padding */
    border: 1px solid #838383;
    border-radius: 2px;
}

/* Focus style for input fields and combo boxes inside info_frame */
#info_frame QLineEdit:focus,
#info_frame QComboBox:focus {
    border-color: #0055ff;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="Imagelabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>10</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI Semibold</family>
               <pointsize>-1</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Select Image</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="uploadButton">
             <property name="text">
              <string>Upload image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <property name="rightMargin">
            <number>30</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Gender</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="radioframe">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QRadioButton" name="maleradioButton">
                    <property name="text">
                     <string>Male</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="femaleradioButton">
                    <property name="text">
                     <string>Female</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FirstnameLIneEdit"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Last Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LastNameLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Student Phone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="studentLineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Grade</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="gradeLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addressLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Parent Phone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="parentLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>BirthDate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="function_frame">
     <property name="styleSheet">
      <string notr="true">/* Style for function_frame QPushButton */
#function_frame QPushButton {
    font-size: 14px;
    padding: 5px 10px;
    border: 2px solid #f0f0f0;
    border-radius: 5px;
    background-color: #84e8f7;
}

/*Style delete btn*/

#function_frame #delete_btn{
background-color: #ff8183;
}

/*Hover Style for buttons insided the function_frame*/
#function_frame QPushButton:hover{
border-color: #4c96f7;
font-size: 15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="add_btn">
        <property name="toolTipDuration">
         <number>-114</number>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>add_icon.svg</normaloff>add_icon.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="upd_btn">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>update_icon.svg</normaloff>update_icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="select_btn">
        <property name="text">
         <string>Select</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>select_Icon.svg</normaloff>select_Icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="search_btn">
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>search_Icon.svg</normaloff>search_Icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>clear_Icon.svg</normaloff>clear_Icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="delete_btn">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>delete_Icons.svg</normaloff>delete_Icons.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section{
border:none;
padding: 3px,3px;
text-align:left;
}

QTableWidget::Item{
color:#000;
padding-left: 7px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="datatableWidget">
        <property name="styleSheet">
         <string notr="true">#datatableWidget {
    background-color: #fff;
    border: none;
    border-radius: 10px;
}</string>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::PenStyle::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>35</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <column>
         <property name="text">
          <string>FirstName</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LastName</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gender</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>StudentPhone</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>BirthDate</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ParentPhone</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Address</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Grade</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Photo</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
